internal/subshell/bash/bash.go

```diff
@@ -45,14 +50,39 @@ func (v *SubShell) SetBinary(binary string) {
 }
 
 // WriteUserEnv - see subshell.SubShell
-func (v *SubShell) WriteUserEnv(cfg sscommon.Configurable, env map[string]string, envType sscommon.EnvData, _ bool) error {
-	homeDir, err := fileutils.HomeDir()
+func (v *SubShell) WriteUserEnv(cfg sscommon.Configurable, env map[string]string, envType sscommon.RcIdentification, _ bool) error {
+	rcFile, err := v.RcFile()
 	if err != nil {
-		return errs.Wrap(err, "HomeDir failed")
+		return errs.Wrap(err, "RcFile failure")
 	}
 
 	env = sscommon.EscapeEnv(env)
-	return sscommon.WriteRcFile("bashrc_append.sh", filepath.Join(homeDir, ".bashrc"), envType, env)
+	return sscommon.WriteRcFile("bashrc_append.sh", rcFile, envType, env)
+}
+
+func (v *SubShell) WriteCompletionScript(completionScript string) error {
+	dir := "/usr/local/etc/bash_completion.d/"
```

Comment:
Are there no directories relative to the HOME directory where we can write to?  This would probably generate an error unless run as a super-user...

Reply:
There's `~/.bash_completion` but that's a single file that's used for ALL completions. From my research I couldn't see it being used much.

Fact is bash completions don't even work unless you install the bash-completion plugin for bash, which I can't imagine is very widely used. It just feels like a waste of effort to do more than what is being done here.